noisemodelling-jdbc/src/test/java/org/noise_planet/noisemodelling/jdbc/AttenuationCnossosTest.java

```diff
@@ -0,0 +1,11002 @@
+                System.out.println("ici");
+            } else {
+                writer.write("     - No\n");
+            }
+
+            if (valLA[0] < 0.1) {
+                writer.write("     - Yes\n"); // With lateral diffraction (Yes)
+            } else {
+                writer.write("     - No\n");
+            }
+            double v = valLA[1];
+            double vLA = Math.round(valLA[0] * 100.0) / 100.0;
+            writer.write("     - " + vLA +" / " + frequencies[(int)v]+ "\n");
+            writer.write("     - `Details <./.build/TC11_D.html>`_\n");
+
+        }catch (IOException e) {
+            System.out.println("Erreur lors de l'écriture dans le fichier : " + e.getMessage());
+        }*/
+
+
+        //Path1 : right lateral
+        double[] expectedWH = new double[]{0.00, 0.00, 0.00, 0.01, 0.08, 0.40, 2.07, 9.99};
+        double[] expectedCfH = new double[]{25.07, 25.45, 26.93, 29.26, 21.08, 5.11, 0.53, 0.10};
+        double[] expectedAGroundH = new double[]{-1.51, -1.51, -1.51, -1.51, -1.51, -1.51, -1.51, -1.51};
+        double[] expectedWF = new double[]{0.00, 0.00, 0.00, 0.01, 0.05, 0.31, 1.60, 7.90};
+        double[] expectedCfF = new double[]{25.05, 25.34, 26.56, 29.11, 23.63, 7.01, 0.74, 0.13};
+        double[] expectedAGroundF = new double[]{-1.51, -1.51, -1.51, -1.51, -1.51, -1.51, -1.51, -1.51};
+
+        expectedAlphaAtm = new double[]{0.12, 0.41, 1.04, 1.93, 3.66, 9.66, 32.77, 116.88};
+        expectedAAtm = new double[]{0.00, 0.01, 0.03, 0.05, 0.10, 0.27, 0.90, 3.22};
+        expectedADiv = new double[]{38.75, 38.75, 38.75, 38.75, 38.75, 38.75, 38.75, 38.75};
+        expectedDeltaDiffSRH = new double[]{14.47, 17.92, 22.26, 26.57, 30.18, 33.37, 36.43, 39.45};
+        expectedLH = new double[]{41.28, 37.82, 33.47, 29.14, 25.48, 22.12, 18.43, 13.09};
+
+        //Actual values
+        proPath = propDataOut.getPropagationPaths().get(1);
+
+        double[] actualWH = proPath.groundAttenuation.wH;
+        double[] actualCfH = proPath.groundAttenuation.cfH;
+        double[] actualAGroundH = proPath.groundAttenuation.aGroundH;
+        double[] actualWF = proPath.groundAttenuation.wF;
+        double[] actualCfF = proPath.groundAttenuation.cfF;
+        double[] actualAGroundF = proPath.groundAttenuation.aGroundF;
+
+        actualAlphaAtm = propDataOut.genericMeteoData.getAlpha_atmo();
+        actualAAtm = proPath.aAtm;
+        actualADiv = proPath.aDiv;
+        actualDeltaDiffSRH = proPath.aBoundaryH.deltaDiffSR;
+        actualLH = addArray(proPath.aGlobalH, SOUND_POWER_LEVELS);
+
+        //Assertions
+        assertDoubleArrayEquals("WH", expectedWH, actualWH, ERROR_EPSILON_LOW);
+        assertDoubleArrayEquals("CfH", expectedCfH, actualCfH, ERROR_EPSILON_LOW);
+        assertDoubleArrayEquals("AGroundH", expectedAGroundH, actualAGroundH, ERROR_EPSILON_LOW);
+        /*assertDoubleArrayEquals("WF", expectedWF, actualWF, ERROR_EPSILON_LOW);
```

Comment:
did not see, some assert are commented. I am working on TC11 as I have different results than you with the patch (related to polygon normalisation?).